tp/src/addressing.rs

```diff
@@ -100,14 +81,6 @@ pub fn make_namespace_registry_address(namespace: &str) -> Result<String, ApplyE
     Ok(NAMESPACE_REGISTRY_PREFIX.to_string() + &hash(prefix, 64)?)
 }
 
-pub fn get_sawtooth_admins_address() -> Result<String, ApplyError> {
```

Comment:
This can also be removed from the sabre cli, currently duplicates this function in cli/src/transactions.rs

Reply:
Done